main.py

```
@@ -186,129 +187,290 @@ async def delete_user(
+        authorization: Annotated[str, Header(description="로그인 토큰")]
 ):
-    return {"post_id": post_id}
+    return Response(status_code=status.HTTP_204_NO_CONTENT)
 
 
 # 댓글 작성 (특정 게시글에 댓글을 작성합니다.)
 @app.post("/posts/{post_id}/comments")
 async def post_comment(
-        post_id: str
+        post_id: Annotated[str, Path(description="게시글 ID")],
+        authorization: Annotated[str, Header(description="로그인 토큰")],
```

Ruff (0.14.13)

Unused function argument: `authorization`

(ARG001)